modules/claims_api/app/models/claims_api/power_of_attorney.rb

```diff
@@ -22,15 +22,23 @@ def date_request_accepted
     end
 
     def representative
-      form_data.merge(current_poa: current_poa, participant_id: nil)
+      form_data.merge(participant_id: nil)
     end
 
     def veteran
       { participant_id: nil }
     end
 
+    def old_poa
```

Comment:
I think a better name for this is previous_poa. Old seems a little off to me.

Reply:
I can change that.

Reply:
Done